README.md

```diff
@@ -34,30 +34,44 @@ create -> launch
 learn  <-  measure
 ```
 
-We want to rapidly iterate our research-first mindset by constantly launching with an MVP implementations for every initiative.  
+#### Practically speaking though?
```

Comment:
I'd say move this to a `###` header called `Setup`.

Reply:
done